src/main.rs

```diff
@@ -76,6 +77,39 @@ fn run() -> Result<(), failure::Error> {
                 .subcommand(
                     SubCommand::with_name("list")
                 )
+                .subcommand(
+                    SubCommand::with_name("write")
+                        .arg(
+                            Arg::with_name("expiration")
+                            .short("e")
+                            .long("expiration")
+                            .takes_value(true)
+                            .value_name("SECONDS")
+                            .help("the time, measured in number of seconds since the UNIX epoch, at which the entries should expire"),
+                        )
+                        .arg(
+                            Arg::with_name("time-to-live")
+                            .short("t")
+                            .long("ttl")
+                            .value_name("SECONDS")
+                            .takes_value(true)
+                            .help("the number of seconds for which the entries should be visible before they expire. At least 60"),
+                        )
+                        .subcommand(
+                            SubCommand::with_name("bulk")
+                                .about("upload multiple key-value pairs at once")
+                                .arg(
+                                    Arg::with_name("id")
+                                        .help("the id of your Workers KV namespace")
+                                        .index(1),
+                                )
+                                .arg(
+                                    Arg::with_name("path")
+                                    .help("the json file of key-value pairs to upload, in form [{\"key\":..., \"value\":...}\"...] OR the directory of files to upload.")
```

Comment:
The json and directory upload are two very distinct things, I wonder if this needs to be broken up into different argument names or even second-level command names. I'm also not sure using the bulk endpoint should be transparent to the user, what if it was just write?

```bash
wrangler kv write --pairs key=val
wrangler kv write -p key0=val0 keyN=valN
wrangler kv write -p '{"key0":"val0", "keyN":"valN"}'

wrangler kv write --directory dir/
```

Reply:
This is definitely a design decision still in flux! A lot of DB clis (redis for example) have separate commands for individual and bulk uploads, so @ashleymichal and I are now considering actually having two commands, like `write` and `write-bulk`. I still think that inferring the type of upload (kv pairs from a file or an entire directory) can be implicitly handled and decided by the type of file passed in to `write-bulk` (file or directory?) 

This design discussion still ongoing, so take what I'm saying right now with a grain of salt!

Reply:
we have a few ideas around where to separate these use cases, we'll be discussing next week with Steve, Ashley, etc.